components/Header.tsx

```diff
@@ -6,46 +6,60 @@ import Link from "next/link";
 import SearchBar from "./SearchBar";
 import { IconMenu } from "@tabler/icons-react";
 import Headroom from "react-headroom";
+import NavbarMobile from "./NavbarMobile";
+import { useState } from "react";
```

Comment:
If you're using `useState` you might need to add `"use client"` otherwise client side rendering is not going to work...